src/app/shared/components/feature-card/feature-card.component.spec.ts

```diff
@@ -1,38 +1,53 @@
-import {async, ComponentFixture, TestBed} from '@angular/core/testing';
-
 import {FeatureCardComponent} from './feature-card.component';
-import {CommonModule} from '@angular/common';
+import {Component} from '@angular/core';
+import {Feature} from '../../models/Feature';
+import {initContext, TestContext} from '../../../../testing/test.context';
 import {MatButtonModule, MatCardModule, MatIconModule, MatSlideToggleModule} from '@angular/material';
-import {CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
 
-describe('FeatureCardComponent', () => {
-  let component: FeatureCardComponent;
-  let fixture: ComponentFixture<FeatureCardComponent>;
+const mockFeature1: Feature = {
```

Comment:
why cant this be within the `TesteeFeatureCardComponent `

Reply:
As I have declared two mocks, I kept it outside the component. If the second mock feature is not needed, I ll place it inside the component